mobile/lib/services/backup.service.dart

```diff
@@ -329,7 +334,9 @@ class BackupService {
         }
 
         if (file != null) {
-          String originalFileName = asset.fileName;
+          String? originalFileName =
+              await _assetMediaRepository.getOriginalFilename(asset.localId!);
+          originalFileName ??= asset.fileName;
```

Comment:
This fallback should maybe be inside the func above?

Is this syntax common in the mobile codebase? I find it really confusing lol.

Reply:
Yeah, if null, assign the value, otherwise keep the current value.

Inside the function, the AssetEntity can be null, so there is no property to get the other filename from